lib/validations.js

```diff
@@ -361,8 +361,8 @@ function validateExclusion(attr, conf, err, options) {
 function validateFormat(attr, conf, err, options) {
   if (nullCheck.call(this, attr, conf, err)) return;
 
-  if (typeof this[attr] === 'string') {
-    if (!this[attr].match(conf['with'])) {
+  if (typeof this[attr] === 'string' || typeof this[attr] === 'number') {
```

Comment:
I prefer we use `conf[with].test(this[attr])` here instead of coercing numbers to string and using `match`. [RegExp test method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/RegExp/test).